addons/sales_team/migrations/9.0.1.0/pre-migration.py

```diff
@@ -1,72 +1,20 @@
-# -*- coding: utf-8 -*-
-##############################################################################
-#
-#    OpenUpgrade module for Odoo
-#    @copyright 2014-Today: Odoo Community Association
-#    @author: Sylvain LE GAL <https://twitter.com/legalsylvain>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from openupgradelib import openupgrade
 
 
-OBSOLETE_VIEWS = (
-    'crm_case_section_salesteams_search',
-    'crm_case_section_salesteams_view_kanban',
-    'crm_case_section_view_form',
-    'crm_case_section_view_tree',
-    'res_user_form',
-    'view_sale_config_settings',
-    'view_users_form_preferences',
-)
-
-
-def recursive_delete(cr):
```

Comment:
we don't need the whole function. Before #509, the upgrade would choke on all kinds of view problems, but with commit https://github.com/OCA/OpenUpgrade/commit/33ff049e18fa7b15892c8dd141f64e8e1fd53773, we don't need any manual cleanup, because the offending views will be dropped later (or changed during the upgrade)